javascript-typing-tutor/script.js

```diff
@@ -0,0 +1,37 @@
+var value = 0;
+var attempts = 0;
+var $text = document.querySelectorAll('span');
+var $score = document.querySelector('p');
+
+document.addEventListener('keydown', function (e) {
+  if (e.key === $text[value].innerText) {
+    $text[value].className = 'green-text';
+    attempts++;
+    value++;
+    $score.innerHTML = 'Score: ' + value + '/27';
+    if (value === 27) {
+      alert('Your typing % is: ' + Math.round(27 / attempts * 100) + ' %');
```

Comment:
Ewwwwwwwwwwwww, no `alert` should ever be present in modern code. You should make a modal for this! Thankfully the process to make one should be fresh in your mind 🤓